src/openms/include/OpenMS/FILTERING/ID/IDFilter.h

```diff
@@ -1156,6 +1156,9 @@ namespace OpenMS
                                  all_peptides);
     }
 
+    /// Filter identifications by "N best" spectra. If a spectrum has a better PSM (PeptideHit) than another it is considered better.
```

Comment:
`The vector is sorted and reduced to @p n elements. If the vector's size 's' is less than @p n, only 's' best spectra are kept.